spec/services/control_rate_service_spec.rb

```diff
@@ -333,7 +333,7 @@ def refresh_views
   end
 
   context "caching" do
-    let(:memory_store) { ActiveSupport::Cache.lookup_store(:redis_store) }
+    let(:memory_store) { ActiveSupport::Cache.lookup_store(:redis_cache_store) }
```

Comment:
👾  I think I wrote that, this should actually be
```suggestion
    let(:memory_store) { ActiveSupport::Cache.lookup_store(:memory_store) }
```